opre_ops/ops_site/fixtures/fake_data.json

```diff
@@ -1,4 +1,22 @@
 [
+{
+    "model": "auth.user", 
+    "pk": 1, 
+    "fields": {
+        "password": "pbkdf2_sha256$260000$SPT55SvTcijon2Y3dU3U5U$gAGQU7lfr+nz+j9kovR8/iGlt1TFAbwuJo7F+dEObeM=", 
```

Comment:
_thought (nonblocking):_ Having the test user in an overall `fake_data.json` works great for now, since we are just developing locally during this sprint cycle. When we have cloud dev and demo environments, we'll want a lightweight layer of access restriction. At that point, we will likely want to move this test user to a separate fixture so that all models except the user model can be loaded in.